public/js/forecast.js

```diff
@@ -0,0 +1,13 @@
+const forecastGetter = async (location) => {
+	const response = location.coords ? await fetch(`/weather?lat=${location.coords.latitude}&lon=${location.coords.longitude}`) : await fetch(`/weather?address=${location}`)
+	const data = await response.json()
+	forecastMarkupCreator(data)
+}
+
+const forecastMarkupCreator = (data) => {
+	if (data.error) return weatherMessage.textContent = data.error
+	weatherIcon.style.display = 'block'
+    weatherIcon.alt = `${data.description}`
+    weatherIcon.src = `${data.icon}`
+    weatherMessage.textContent = `You're in ${data.location}. The temperature is ${data.temperature}. And it feels like ${data.feelslike}.`
+}
```

Comment:
end of line 
I would suggest to install the prettier and run it.
you can copy the config from other projects (my branches)
```bash
$ npx prettier -w
```

Reply:
Fixed.